tests/chart/test_auth_sidecar.py

```diff
@@ -147,6 +146,52 @@ def test_auth_sidecar_resources_with_dag_server_enabled(self, kube_version):
             ],
         )
 
-        common_dagserver_sts_test_cases(docs, 1)
+        common_sts_test_cases(docs, 1, "release-name-dag-server", "StatefulSet")
```

Comment:
This applies to more than just `sts`
```suggestion
        common_pod_manager_test_cases(docs, 1, "release-name-dag-server", "StatefulSet")
```